html.c

```diff
@@ -638,7 +640,13 @@ rndr_raw_block(struct lowdown_buf *ob,
 
 	if (!hbuf_put(ob, param->text.data + org, sz - org))
 		return 0;
-	return hbuf_putc(ob, '\n');
+
+	result = hbuf_putc(ob, '\n');
+
+	if (!hbuf_putb(ob, content))
```

Comment:
Print out the inner content. Without this change, any child nodes of `LOWDOWN_BLOCKHTML` are passed over.